api/validation/api.go

```diff
@@ -41,17 +41,15 @@ type CheckRepresentation struct {
 
 // Parameter references
 const (
-	prmUserID                   = "user_id"
-	prmUserGender               = "user_gender"
-	prmUserFirstName            = "user_firstName"
-	prmUserLastName             = "user_lastName"
-	prmUserEmail                = "user_emailAddress"
-	prmUserPhoneNumber          = "user_phoneNumber"
-	prmUserBirthDate            = "user_birthDate"
-	prmUserBirthLocation        = "user_birthLocation"
-	prmUserIDDocumentType       = "user_idDocType"
-	prmUserIDDocumentNumber     = "user_idDocNumber"
-	prmUserIDDocumentExpiration = "user_idDocExpiration"
```

Comment:
why idDocExpiration is removed ?

Reply:
Not used. We don't have check on this field